mountpoint-s3/src/main.rs

```diff
@@ -107,6 +107,9 @@ struct CliArgs {
     #[clap(long, help = "Force path-style addressing")]
     pub path_addressing: bool,
 
+    #[clap(long, help = "Value of the 'x-amz-request-payer' header sent to S3")]
```

Comment:
Update help text a bit, should be something like "Enable accessing to requester pays bucket".